envs/gke/storage-class.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2019, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation.
+
+# Wazuh StorageClass
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: wazuh-storage
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-balanced
+  fstype: ext4
+volumeBindingMode: WaitForFirstConsumer
+reclaimPolicy: Delete  # Change to retain, useful in case you delete the PersistentVolumeClaim
```

Comment:
The deployment of this repository is aimed at environments that can be productive, so it is necessary to have the retain option by default in the reclaimPolicy option